resources/scss/components/_global.scss

```diff
@@ -46,22 +46,32 @@ h1 {
 
 h2 {
     @apply .text-3xl;
+
+    margin-bottom: 0.5rem;
```

Comment:
Is there an existing tailwind class that is this same spacing we could utilize here in an `@apply` instead of a direct number in the CSS?

Reply:
I had her type these in explicitly. We will convert them.

Reply:
@AmbyrElan can you add ml-2 to line 48 and all the other headings,
`@apply .text-3xl .ml-2;` 
And for h3 through h6 
Add ml-1 in the same manner. 
Sorry about that!